docs/content/docs/dev/table/sql/queries/window-tvf.md

```diff
@@ -289,6 +289,100 @@ Here is an example invocation on the Bid table:
 +------------------+------------------+-------+
 ```
 
+### SESSION
+
+{{< hint info >}}
+Note:
+1. Session Window TVF is not supported in Batch mode now.
+2. Session Window Aggregation is not supported [Performance Tuning]({{< ref "docs/dev/table/tuning" >}}) now.
```

Comment:
```suggestion
2. Session Window Aggregation is not supported by local-global optimization. You can refer to  [Performance Tuning]({{< ref "docs/dev/table/tuning" >}}) for more details.
```

Reply:
My original intention is that all optimizations involved in `Performance Tuning` are not supported. What about 
```
Session Window Aggregation is not supported all optimizations in [Performance Tuning]({{< ref "docs/dev/table/tuning" >}}) now.
```